recipes/calcam/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "calcam" %}
+{% set version = "2.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/calcam-{{ version }}.tar.gz
+  sha256: ec37dae2d7bb9bf4a6a8e110328bcafb6f4dbc8d2906b856daccff04551bf2a3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - calcam = calcam:start_gui
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.5
+    - matplotlib-base
+    - opencv
+    - pyqt
+    - scipy
+    - vtk >=7,!=9.1.*
+  run_constrained:
+    - libstdcxx-ng >=13
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
`calcam` has a dependency of `opencv-python-headless` which does not exist in conda-forge channel.
So, pip check throw the missing error.
Let me reverse this change and replace PyPi's source URL with GitHub's one.

Reply:
Can you please add a patch then that removes/rewrites this dependency from the package metadata?